content/about.md

```diff
@@ -42,6 +36,9 @@ Rob started his software career as a web developer in healthcare. He now writes
 {{< person name="Wes Hinsley" img="wes.jpg" url="https://github.com/weshinsley">}}
 Wes is a computer engineer and scientist, with a Ph.D making tools for ocean modelling, and then moving to modelling epidemics. He enjoys problem solving, Advent Of Code, (especially on 40-year old hardware), and making interactive and graphical teaching games. By night, he plays music and squash, and makes very good tea.
 {{< /person >}}
+{{< person name="Mantra Kusumgar" img="quokka.webp" url="https://github.com/weshinsley">}}
```

Comment:
Do you want to link to yourself? And should we create a follow on ticket for using a photo of yourself (though the quokka is v cute)